internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -367,6 +364,20 @@ func (provisioner *KopsProvisioner) ProvisionCluster(cluster *model.Cluster, aws
 		},
 	}
 
+	if len(cluster.ProvisionerMetadataKops.Networking) > 0 && (cluster.ProvisionerMetadataKops.Networking == "amazonvpc" || cluster.ProvisionerMetadataKops.Networking == "weave") {
+		files = append(files, k8s.ManifestFile{
+			Path:            "manifests/calico-policy-only-aws-cni.yaml",
+			DeployNamespace: "kube-system",
+		})
+	}
+
+	if len(cluster.ProvisionerMetadataKops.Networking) > 0 && cluster.ProvisionerMetadataKops.Networking == "calico" {
```

Comment:
```
if len(cluster.ProvisionerMetadataKops.Networking) > 0 && cluster.ProvisionerMetadataKops.Networking == "calico" {
		files = append(files, k8s.ManifestFile{
			Path:            "manifests/calico-cni.yaml",
			DeployNamespace: "kube-system",
		})
	} else {
        files = append(files, k8s.ManifestFile{
			Path:            "manifests/calico-policy-only-aws-cni.yaml",
			DeployNamespace: "kube-system",
		})
}
```
as we want this path `Path: "manifests/calico-policy-only-aws-cni.yaml",` for anything other then `calico`

Reply:
will be good if we can also rename `manifests/calico-policy-only-aws-cni.yaml` to something like `manifests/calico-network-policy-only.yaml",`